js/currency-converter.js

```diff
@@ -0,0 +1,381 @@
+/*For ES LINT Purposes */
+if(conversion_data == undefined){
+    var conversion_data = {};
+}
+
+$(function () {
+    show_inputs_relevant_to_selected_year();
+});
+
+function show_inputs_relevant_to_selected_year() {
+    if (get_currency_year() > 1970) {
+        /* Hide old UK currency inputs & show modern */
+
+        $(".newer-currencies").each(function () {
+            $(this).show();
+            $(this).find("input").attr('required', true);
+        });
+
+        $(".older-currencies").each(function () {
+            $(this).hide();
+            $(this).find("input").attr('required', false);
+        });
+    }
+    else if (get_currency_year() <= 1970) {
+        /* Hide modern UK currency inputs & show old */
+
+        $(".newer-currencies").each(function () {
+            $(this).hide();
+            $(this).find("input").attr('required', false);
+        });
+
+        $(".older-currencies").each(function () {
+            $(this).show();
+            $(this).find("input").attr('required', true);
+        });
+    }
+}
+
+$("#currency-year").change(function () {
+    change_fieldset_text();
+    show_inputs_relevant_to_selected_year();
+
+});
+
+function change_fieldset_text() {
+    $('#currency-legend').text("Enter currency to show its purchasing power in " + get_currency_year());
+    $('#currency-submit').val("Show purchasing power in " + get_currency_year());
+}
+
+function build_blockquote(intro, century) {
+    return "<div id='currency-century-intro'>"+ "<p>" + intro + "</p>" +
+        "<p id='currency-century-intro-read-more'><a href='./" + century + "-century.php'>Read more about the " + century + " century. </a></p></div>";
+}
+
+function currency_output() {
+    var user_inputs = get_user_inputs();
+    var currency_formula_return_values = currency_formula();
+
+    var century_preview = "";
+    var converted_money_string = currency_formula_return_values.money.toLocaleString('en-GB', {
```

Comment:
Hi @AshTNA - I'm afraid the locales parameters toLocaleString aren't supported in IE10 or below.

Reply:
Hi @gtvj - I have written a prototype function to try and handle this. It currently works up to the number 999,999. If you are happy with how I have written it then I will try implement something that works up to maybe 999,999,999,999. I have added QUnit tests for this function and they seem to be passing OK.